Utils/release_notes_generator.py

```diff
@@ -283,8 +291,12 @@ def aggregate_release_notes(pack_name: str, pack_versions_dict: dict, pack_metad
 
     """
     pack_release_notes, latest_version = merge_version_blocks(pack_versions_dict)
-    partner = ' (Partner Supported)' if is_partner_supported_in_metadata(pack_metadata) else ''
-    return (f'### {pack_name} Pack v{latest_version}{partner}\n'
+    pack_version_title = latest_version
+    if is_partner_supported_in_metadata(pack_metadata):
+        pack_version_title += ' (Partner Supported)'
+    elif is_community_supported_in_metadata(pack_metadata):
+        pack_version_title += ' (Community Supported)'
```

Comment:
@DeanArbel the string here is wrong, it should be `Community Contributed`

Reply:
will open another PR